app/src/main/java/com/vero/tasky/agenda/data/worker/SaveEventWorker.kt

```diff
@@ -56,21 +62,25 @@ class SaveEventWorker @AssistedInject constructor(
             return Result.failure()
         }
 
+        var multipartPhotoFiles : List<File>? = null
```

Comment:
Personally I tend to never make lists null
I always make them empty

Reply:
Since you tend to always run a forEach.. if its empty, they never run
So its the same as having it null, but you remove the nullability